### DISCUSSION TITLE
Nested fields

### DISCUSSION THREAD

**Discussion by jeremiah-blessing** [Budibase/budibase#2753] [Help] [answered]
We are creating a POC CRUD application for one of our internal tools. I can see Budibase supports some field types (text, number, boolean, DateTime). 

Is it possible to create a CRUD application with API having nested values?

Eg

```
{
users : [
 {
  name : "name 1",
  age : 26
 },
 {
  name : "name 2",
  age : 31
 }
],
usersCount : 10,
company : "company name"
}
```

**Comment by mjashanks** [comment, marked as answer]
Hi @jeremiah-blessing  - this is not possible at the moment, but will be by the end of October. We will be adding the concept of "Transformers", whereby you can write Javascript to transform the JSON response before it's used by Budibase.

**Reply by mjashanks** [reply]
you can follow this here: https://github.com/Budibase/budibase/issues/2093.

**Reply by jeremiah-blessing** [reply]
@mjashanks Sorry for the delayed response. Thank you so much. We will be looking for the 'Transformers' update.
